src/components/Breakpoints/Breakpoints.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import cssVariables from "../../helpers/CSSVariablesHelper";
+import getCSSVariable from "../../helpers/getCSSVariable";
+import Heading from "../Heading/Heading";
+import { CSSVariablesInterface } from "../../interfaces";
+import { getCategory } from "../../utils/componentCategories";
+import * as stories from "./Breakpoints.stories.tsx";
+
+<Meta
+  title={getCategory("Breakpoints")}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=10975%3A16",
+    },
+  }}
+/>
+
+# Breakpoints
+
+The DS adheres to four global breakpoints for responsive layouts and typography.
+
+- Desktop: 1280px
+- Tablet Landscape: 960px
+- Tablet Portrait: 600px
+- Mobile: 320px
+
+The following CSS variables are available through the DS.
+
+```
+--breakpoint-xl: 1280px;
+--breakpoint-large: 960px;
+--breakpoint-medium: 600px;
+--breakpoint-small: 320px;
+```
+
+<!-- <Story story={stories.breakpoints} /> -->
```

Comment:
Are you still trying to do this? If not, this page can just be a documentation page and not an interactive page (which is fine). Then you can remove the other `Breakpoints.stories.tsx` file. This does mean that for new breakpoints this would have to be updated manually but I don't think that's a big deal.

Reply:
@EdwinGuzman I was able to get the dynamic inclusion of the classes in the `.mdx` story to work and more or less formatted how I wanted, but the look and feel was different enough that I think I am going to give up on it.  I was intially concerned that losing this functionality would be a problem, but in the end, like you, I feel it is not a big deal to update manually.  I will rmeove this and the `.tsx` file.